task_11/index.html

```diff
@@ -0,0 +1,292 @@
+                    <label class="document-type-radio">
+                        <input type="radio" name="document-type" value="social" required>
+                        <span class="select-wrapper">
+                            Social Account
+                            <img src="img/social.png" alt="">
+                        </span>
+                    </label>
+                </div>
+            </section>
+            
+            <section class="file-upload counter">
+                <h2>Upload your documents <span>(Doc, Docx, Text, PFD, Markdown)</span></h2>
+                <label class="file-upload-btn">
+                    <input type="file" name="file" accept=".doc, .docx, .txt, .pdf, .md" multiple required hidden>
+                    <span class="select-wrapper">
+                        <span><i class="fas fa-cloud-upload-alt"></i>Upload a new file</span>
+                    </span>
+                </label>
+                <div class="file-upload-table">
+                    <table>
+                        <tr>
+                            <th>File name</th>
+                            <th>File size</th>
+                            <th>Number of words</th>
+                            <th>Remove</th>
+                        </tr>
+                        <tr>
+                            <td><i class="fas fa-file-alt"></i>busines_operations.docx</td>
+                            <td>78kb</td>
+                            <td>842</td>
+                            <td><i class="fas fa-trash"></i></td>
+                        </tr>
+                        <tr>
+                            <td><i class="fas fa-file-alt"></i>busines_financial.docx</td>
+                            <td>26kb</td>
+                            <td>2,326</td>
+                            <td><i class="fas fa-trash"></i></td>
+                        </tr>
+                    </table>
+                </div>
+            </section>
+            
+            <section class="language-select counter">
+                <h2>Choose your languages pairings</h2>
+                <div class="src-language-select">
+                    <label for="src-language">
+                        <span>From</span>
+                    </label>
+                    <select id="src-language" name="src-language">
+                        <option value="arabic">Arabic</option>
+                        <option value="english" selected>English</option>
+                        <option value="kurdish">Kurdish</option>
+                        <option value="ukrainian">Ukrainian</option>
+                    </select>
+                </div>
+                <div class="res-language-select">
+                    <label for="res-language-1">
+                        <span>To</span>
+                    </label>
+                    <select id="res-language-1" name="res-language-1">
+                        <option value="arabic" selected>Arabic</option>
```

Comment:
In general, servers and APIs use [ISO language codes](https://www.loc.gov/standards/iso639-2/php/code_list.php) as actual values, instead of names. It's easy to parse on a server side.